src/main/java/sigma/task/Task.java

```diff
@@ -0,0 +1,74 @@
+package sigma.task;
+
+/**
+ * Represents a task with a description and a status.
+ */
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor for a Task.
+     *
+     * @param description the description of the task
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Marks this task as done.
+     */
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    /**
+     * Marks this task as not done.
+     */
+    public void markAsNotDone() {
```

Comment:
Perhaps using setNotDone will adhere to setters naming conventions for boolean setters :)